### PR TITLE
Adds DRF 314

### PR DESCRIPTION
**Pull request by dennys-bd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ The first build will take a while.
...
 
+## Adding a new version
+
+1. Add the version to `rest_framework_ccbv.config.py::REST_FRAMEWORK_VERSIONS`
+2. In `build.ini`:
+    - Add the version to envlist
+    - Add the deps to the version (check deps314)
+    - Add the index generation (check testenv:drf314)
+    - Add the doc generation (check testenv:drfbuild314)
+
```
Comment (dennys-bd) [inline README.md]: Not sure if this is the whole deal

Comment (fjsj) [reply README.md]: Double-check here: https://github.com/vintasoftware/classy-django-rest-framework/pull/31/files

File: build.ini [hunk 2]
```diff
@@ -60,6 +60,10 @@ deps313 =
     django==4.0.4
     djangorestframework>=3.13,<3.14
 
+deps314 =
+    django==4.1.0
```
Comment (fjsj) [inline build.ini:64]: Is this the lowest version supported by DRF 3.15?

Comment (fjsj) [reply build.ini:64]: I suppose we should use either lowest or highest at the time Django versions for each DRF version...

**Review by fjsj — APPROVED** [review]
LGTM, but please consider comments

File: build.ini (hunk 2 above)
Comment (dennys-bd) [reply build.ini:64]: image

The previous deps were considering the highest, and they recommend the same

**Pull request merged**